utils/get-lines.ts

```diff
@@ -0,0 +1,10 @@
+export function getLines(contents: string, range?: number[]) {
+  if (!range) {
+    return '';
+  }
+
+  const lines = contents.split('\n');
+  const editableLines =
+    !range || range[1] <= range[0] ? [] : lines.slice(range[0], range[1] - 1);
```

Comment:
Due to the first `if`, the `!range` condition is never.

```suggestion
  if (!range) {
    return '';
  }

  const lines = contents.split('\n');
  const editableLines =
    range[1] <= range[0] ? [] : lines.slice(range[0], range[1] - 1);
```